ci-operator/step-registry/openshift/e2e/aws/ccm/openshift-e2e-aws-ccm-workflow.yaml

```diff
@@ -2,18 +2,31 @@ workflow:
   as: openshift-e2e-aws-ccm
   steps:
     pre:
-    - chain: ipi-aws-pre
+    - ref: ipi-conf
+    - ref: ipi-conf-aws
+    # TODO: add - ref: ipi-install-monitoringpvc
+    # We do not include monitoring PVC introduced in https://github.com/openshift/release/pull/15040
+    # as functionality of this PVC depends on successfull migration to CSI we are currently unable to
+    # fully determine. 
+    - chain: ipi-install
     - ref: ccm-conf-apply-feature-gate
+    - ref: storage-conf-csi-aws-ebs
+    - ref: storage-conf-wait-for-csi-migration
     test:
     - ref: openshift-e2e-test
     post:
     - chain: gather-core-dump
     - chain: ipi-aws-post
     env:
+      TEST_INSTALL_CSI_DRIVERS: aws-ebs
+      TEST_CSI_DRIVER_MANIFEST: manifest-aws-ebs.yaml
+      MCO_IMAGE_OVERRIDE: "quay.io/dgrigore/machine-config-operator:ccm"
+      KCMO_IMAGE_OVERRIDE: "quay.io/dgrigore/cluster-kube-controller-manager-operator:ccm"
       TEST_SKIPS: >-
         \[sig-arch\]\[Early\] Managed cluster should start all core operators \[Skipped:Disconnected\]\|
         \[sig-instrumentation\] Prometheus when installed on the cluster shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured \[Early\] \[Skipped:Disconnected\]\|
-        Alerts shouldn't report any alerts in firing or pending state
+        Alerts shouldn't report any alerts in firing or pending state\|
+        Prometheus when installed on the cluster shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured
```

Comment:
What kind of alerts do we get during the migration? Are these acceptable in prod?

Reply:
That the cluster has unmanaged operators from CVO and that it is unupgradable due to CustromNoUpgrade feature gate present. They are acceptable.